src/auth/AuthContext.tsx

```diff
@@ -150,7 +150,7 @@ export default function AuthContextProvider({children}: PropsWithChildren<{}>) {
     async function signInWithPhoneNumber(phoneNumber: string) {
       try {
         const confirmationHandler = await auth().signInWithPhoneNumber(
-          '+47' + phoneNumber,
+          phoneNumber,
```

Comment:
@rosvik Jeg skjønner ikke koden godt nok, men ingen behov for en prefix her? Tenkte siden tidligere "+47" + phoneNumber sendte inn "+47xxxxxxxx", så vil nå også prefix være inkludert i phoneNumber?

Reply:
Fordi jeg sender `phoneValidation.phoneNumber` som input, følger prefiks med i `phoneNumber`. Kan muligens oppdatere variabelnavnet for å gjøre det tydligere? `phoneNumberWithPrefix`?

Reply:
Nice. Du kan avgjøre. Ikke meg som utvikler 🙈